gradle.properties

```diff
@@ -9,6 +9,7 @@ project_vcs=https://github.com/melix/jmh-gradle-plugin.git
 jacocoVersion      = 0.8.10
 jmhVersion         = 1.37
 shadowVersion      = 7.1.2
+shadowForkVersion  = 8.1.2
```

Comment:
```suggestion
shadowForkVersion  = 8.1.3
```

Reply:
Ah, nice. 8.1.3 wasn't on [Gradle Plugins](https://plugins.gradle.org/plugin/io.github.goooler.shadow) when I opened the PR.

Reply:
Yeah, cause I unified this group to `io.github.goooler.shadow` and published it to MavenCentral.